.github/workflows-source/schedule-daily.yml

```diff
@@ -153,7 +153,7 @@ jobs:
         shell: bash
         run: |
           set -xeuo pipefail
-          echo "BENCHMARK_TARGETS=$(bazel query 'attr(tags, system_test_benchmark, //rs/...)' | tr '\n' ' ')" >> $GITHUB_ENV
+          echo "BENCHMARK_TARGETS=$(bazel query 'attr(tags, system_test_benchmark, //rs/...)' | grep -v head_nns | tr '\n' ' ')" >> $GITHUB_ENV
```

Comment:
is this on purpose?

Reply:
Yes. We currently always run those benchmarks twice, which we don't need. This change excludes the duplicates, currently the ones with `HEAD` NNS because in those tests we don't really care, but we can change this later if needed.